### PR TITLE
add featured label on citizenproject header

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: tests/Controller/EnMarche/CitizenProjectControllerTest.php [hunk 1]
```diff
@@ -343,6 +343,31 @@ public function testAuthenticatedAdherentCanFollowCitizenProject()
         $this->assertFalse($this->seeRegisterLink($crawler, 0));
     }
 
+    public function testFeaturedCitizenProject()
+    {
+        /** @var CitizenProject $citizenProject */
+        $citizenProject = $this->getCitizenProjectRepository()->findOneByUuid(LoadCitizenProjectData::CITIZEN_PROJECT_1_UUID);
+
+        $this->assertFalse($citizenProject->isFeatured());
```
Comment (HeahDude) [inline tests/Controller/EnMarche/CitizenProjectControllerTest.php]: Assertions can be joined but should be surrounded by empty lines to distinguish them from the logic code.

File: tests/Controller/EnMarche/CitizenProjectControllerTest.php [hunk 2]
```diff
@@ -343,6 +343,31 @@ public function testAuthenticatedAdherentCanFollowCitizenProject()
...
+        $this->manager->flush();
+        $this->manager->clear();
+
+        $citizenProject = $this->getCitizenProjectRepository()->findOneByUuid(LoadCitizenProjectData::CITIZEN_PROJECT_1_UUID);
+
+        $crawler = $this->client->request(Request::METHOD_GET, $citizenProjectUrl);
+
+        $this->assertTrue($citizenProject->isFeatured());
+        $this->assertSame(1, $crawler->filter('.citizen_project_featured')->count());
+        $this->assertContains('Nos coups de coeur', $crawler->filter('.citizen_project_featured')->text());
```
Comment (HeahDude) [inline tests/Controller/EnMarche/CitizenProjectControllerTest.php]: `assertSame` + `trim`

File: tests/Controller/EnMarche/CitizenProjectControllerTest.php [hunk 3]
```diff
@@ -343,6 +343,30 @@ public function testAuthenticatedAdherentCanFollowCitizenProject()
         $this->assertFalse($this->seeRegisterLink($crawler, 0));
     }
 
+    public function testFeaturedCitizenProject()
+    {
+        /** @var CitizenProject $citizenProject */
+        $citizenProject = $this->getCitizenProjectRepository()->findOneByUuid(LoadCitizenProjectData::CITIZEN_PROJECT_1_UUID);
+        $citizenProjectUrl = sprintf('/projets-citoyens/%s', 'le-projet-citoyen-a-paris-8');
```
Comment (HeahDude) [inline tests/Controller/EnMarche/CitizenProjectControllerTest.php]: Why using `sprintf` here?

**Comment by B-Galati** [comment]
Looks like a rebase is needed no ?

File: tests/Controller/EnMarche/CitizenProjectControllerTest.php (hunk 3 above)
Comment (NyKo24) [reply tests/Controller/EnMarche/CitizenProjectControllerTest.php]: habit

Comment (HeahDude) [reply tests/Controller/EnMarche/CitizenProjectControllerTest.php]: So you can remove it :).

**Pull request merged**